trackbasket_be/resources/volunteer.py

```diff
@@ -1,11 +1,17 @@
 from flask_restful import Resource, reqparse, request
+from models.volunteer import Volunteer, db
+from models.basemodel import BaseModel
 
-class Volunteer(Resource):
+class Volunteers(Resource):
 
   def get(self,id):
-    return {'volunteer': id}
+    volunteer = db.session.query(Volunteer).get(id)
+    return {'volunteer': volunteer.name}
 
-  def post(self,id):
+  def post(self, id):
     request_data = request.json
+    volunteer = Volunteer(name=request_data['name'], phone_number=request_data['phone_number'])
+    db.session.add(volunteer)
+    db.session.commit()
```

Comment:
it's looking nice, we need to change this to make sure we write the id passed by the front end to the Volunteers table